src/constants/constants.ts

```diff
@@ -5,4 +5,12 @@ export const COMPLETED = 'COMPLETED';
 export const DONE = 'DONE';
 export const AVAILABLE = 'AVAILABLE';
 export const UNASSINGED = 'UNASSINGED';
+export const STANDUP_ALREADY_SUBMITTED =
+    'Standup already submitted for the day';
+export const dateFormat = 'MMMM DD, YYYY ';
```

Comment:
why in lower case for `dateFormat` alone?

Reply:
fixed